net.go

```diff
@@ -82,6 +82,12 @@ type ping struct {
 	// the intended recipient. This is to protect again an agent
 	// restart with a new name.
 	Node string
+
+	// YouShouldRefute is a courtesy flag that's sent during a probe to
+	// tell the node being probed that it's currently in the suspect or dead
+	// state from the perspective of the prober so it can start right away
+	// trying to refute that.
+	YouShouldRefute bool
```

Comment:
A cleaner way to do this might be to just staple a suspect message and do a compound message.